### PR TITLE
Adds chromosome effects to Cryokinesis, Shocktouch, Chameleon, and Webbing mutations.

### PR DESCRIPTION
**Pull request by moocowswag**
## About The Pull Request
https://www.youtube.com/watch?v=CPKeEeBNSH0
Cryokinesis instability up by 10 (from 20 to 30)
Cryokinesis base cooldown up by 1 second (from 15 to 16) now accepts energetic chromosomes (8 seconds when applied)

Shock touch instability up by 5 (from 30 to 35) now accepts both energetic and power chromosomes 

Shock touch cooldown up by 2 (from 10 to 12) (cooldown of 6 when energetic is applied)

If a power chromosome is applied Shock touch does a weak 7500 power (I`ve seen it hit from 5-7 damage) tesla shock with range of 7 when used on a viable target (extra damage does not apply to target, no you cant use it on yourself, no you cant use it on someone that is shock immune, yes you have to put yourself in harms way to apply the tesla shock.)

(The vars for the tesla shock can be edited by admins for events/abuse)

Chameleon now accepts power chromosomes to fully stealth about 3 seconds faster.

Webbing now accepts energetic chromosomes 

## Why It's Good For The Game
1. Geneticists needs something to do after they find mutations and chromosome farming can greatly expand the time that geneticists spend doing genetics.

2. Only 1 beneficial mutation uses power and energetic chromosomes and that is firebreath, (and arguably void magnet but nobody else uses that no matter how hard I try to get people to try it)

3. We have a whole chromosome system that most mutations cant even use and several mutations are unused because they are simply too weak or irrelevant for anyone to give a damn about them.

This pr kills 3 birds with one stone, geneticists will have more to do, the chromosome system will see more use, and the underused mutations will see more use rather than being bloat.
## Changelog

:cl:
add: Cryokinesis, Shocktouch, Chameleon, and Webbing mutations now have a higher potential
balance: Cryokinesis and Shock touch appear to be weaker and more unstable by default
/:cl:

### CODE REVIEW

**Comment by Shadowflame909** [comment]
genetics mains eating with more job content.

love when genetics gets more love

**Review by MrMelbert — CHANGES_REQUESTED** [review]

**Comment by moocowswag** [comment]
Testing changes, also upping tesla shock power from 5,000 to 7,500 and testing that because it was consistently hitting for 3 damage when its intended to more consistently hit for around 6 and people kept saying it was too weak.

**Comment by moocowswag** [comment]
Shock touch continues to update properly with addition and removal of power chromosome.

**Comment by moocowswag** [comment]
Webbing properly updates, shocktouch still does too.

File: code/datums/mutations/touch.dm [hunk 1]
```diff
@@ -7,17 +7,41 @@
...
 	spell_requirements = NONE
 
+	var/chain = 0
+	//Vars for zaps made when power chromosome is applied, ripped and toned down from reactive tesla armor code.
+	//damage should do about 1 per limb
+	var/zap_power = 5000
+	//range
+	var/zap_range = 7
+	//flags
+	var/zap_flags = ZAP_MOB_DAMAGE
```
[the remark this answers is not in the document]
Comment (moocowswag) [reply code/datums/mutations/touch.dm]: Auto-doc as in?

[the remark this answers is not in the document]
Comment (Sealed101) [reply code/datums/mutations/touch.dm]: `///text goes here`
this is an autodoc

**Comment by moocowswag** [comment]
pass checks plz <3

**Comment by moocowswag** [comment]
tested after rebase, it all still seems to work.

**Pull request merged**